src/Build/BackEnd/Node/OutOfProcNode.cs

```diff
@@ -710,10 +709,8 @@ private void HandleNodeConfiguration(NodeConfiguration configuration)
             }
             catch (DirectoryNotFoundException)
             {
-#if FEATURE_ENVIRONMENT_SYSTEMDIRECTORY
                 // Somehow the startup directory vanished. This can happen if build was started from a USB Key and it was removed.
                 NativeMethodsShared.SetCurrentDirectory(Environment.SystemDirectory);
-#endif
```

Comment:
Why the inconsistency with the other changes? Not that it matters, right?

Reply:
No good reason.